thrust/detail/config/exec_check_disable.h

```diff
@@ -28,7 +28,7 @@
 #if defined(__CUDACC__) && !defined(__NVCOMPILER_CUDA__) && \
     !(defined(__CUDA__) && defined(__clang__))
 
-#define __thrust_exec_check_disable__ #pragma nv_exec_check_disable
+#define __thrust_exec_check_disable__ _Pragma("nv_exec_check_disable")
```

Comment:
Yeah, I looked at this recently and wondered whether the `#pragma` form is kosher. Is this something that actually broke _somewhere_, or just a stylistic fix to make this actually behave more like a declaration annotation than a directive?

Reply:
It didn't break anything for me *yet*, but `clang-format` likes to reformat code from

```
__thrust_exec_check_disable__
__host__ __device__
void foo()
{
  // ...
}
```

to

```
__thrust_exec_check_disable__ __host__ __device__ void foo()
{
  // ...
}
```

which would consume the `__host__ __device__ void foo()` bit as part of the `#pragma`.